ydb/public/sdk/cpp/client/ydb_query/tx.h

```diff
@@ -63,12 +63,16 @@ struct TTxSettings {
         TS_SNAPSHOT_RO
     };
 
-    const ETransactionMode Mode_;
```

Comment:
This may break backward compatibility, users can already use Mode_ public member ih their code